projects/plugins/social/src/js/store/selectors/site-data.js

```diff
@@ -0,0 +1,7 @@
+const siteDataSelectors = {
+	getAPIRootUrl: state => state.siteData?.apiRoot ?? null,
+	getAPINonce: state => state.siteData?.apiNonce ?? null,
+	getRegistrationNonce: state => state.siteData?.registrationNonce ?? null,
```

Comment:
Similar thoughts, on the `null` default here, but we can probably get rid of these as it's only used by the `ConnectScreen` and the values are taken directly from the initial state.

Reply:
I'm wondering if there's some wisdom in passing all data from `initialState` to the store anyway. It would allow to us to get all data and state from the store, rather than sometimes getting it from the store and sometimes from a global window variable. What do you think?

Reply:
I left the `null`s as they're a value in a way that `undefined` is not, but they're probably not necessary. Swapped to using the store in [b00734e](https://github.com/Automattic/jetpack/pull/24203/commits/b00734ed245171a0e60dc37d273ba20aea6ec487)